openshift.spec

```diff
@@ -2,13 +2,13 @@
 %global debug_package %{nil}
 %global gopath      %{_datadir}/gocode
 %global import_path github.com/openshift/origin
-%global commit      d3f40eafae8ae7bbca61981e33f384375307fafa
+%global commit      21fb40637c4e3507cca1fcab6c4d56b06950a149
```

Comment:
I asked Clayton if he was ok with the version embedded in the binary being the Release string, (ie: 2.git.0.c75cb10.el7) but he asked that it only be a git hash.

We can either do some string manipulation on Release to extract the git hash or we can add custom tito build and tag scripts which would cover both --test builds and tag operations. I'll look into that a bit more today.